examples/ccxt.pro/py/build-ohlcv-many-symbols.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+import asyncio
+import ccxtpro
+
+async def loop(exchange, symbol, timeframe):
+    while True:
+        try:
+            trades = await exchange.watch_trades(symbol)
+            if len(trades) > 0:
+                ohlcvc = exchange.build_ohlcvc(trades, timeframe)
+                print("Symbol:", symbol, "timeframe:", timeframe)
+                print ('-----------------------------------------------------------')
+                print (ohlcvc)
```

Comment:
Please, add to the output the iso8601 (human-readable) datetime, the exchange id, the symbol, the timeframe, and the table function here:

- https://github.com/ccxt/ccxt/blob/master/examples/ccxt.pro/py/bitmex_watch_ticker_and_ohlcv.py#L6
- https://github.com/ccxt/ccxt/blob/master/examples/ccxt.pro/py/bitmex_watch_ticker_and_ohlcv.py#L45